### DISCUSSION TITLE
Info Button isn't showed

### DISCUSSION THREAD

**Discussion by thanhtruongnhu** [chatscope/chat-ui-kit-react#85] [Q&A]
Hi everyone,

I try to use `InfoButton` in the following code:
`<ConversationHeader.Actions>
            <VoiceCallButton />
            <VideoCallButton />
            <InfoButton />
          </ConversationHeader.Actions>`

However, the InfoButton did not render for some reasons. Anyone know how to fix this? Thank you.

PS: I already imported `import { InfoButton} from "@chatscope/chat-ui-kit-react";`

**Comment by supersnager** [comment]
@thanhtruongnhu It works for me. Please prepare a minimal repro. 

It also works here: https://chatscope.io/storybook/react/?path=/docs/components-conversationheader--full-featured-with-actions
